### PR TITLE
Update maven publish gradle plugin

### PR DESCRIPTION
**Pull request by tso**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -173,9 +173,8 @@ subprojects {
     apply(from = file("$rootDir/hooks.gradle"))
   }
 
-  if (!path.startsWith(":samples")) {
+  if (!path.startsWith(":samples") && !path.startsWith(":misk-bom")) {
     apply(plugin = "com.vanniktech.maven.publish")
-    apply(from = "$rootDir/gradle-mvn-publish.gradle")
```
Comment (tso) [inline build.gradle.kts:178]: Given I'm removing this is there a point to this file??

Comment (staktrace) [reply build.gradle.kts:178]: Why are you removing this? Won't it break the open-source publication to sonatype? (I'm just guessing, I don't know how publishing of misk works)

**Review by staktrace — APPROVED** [review]
Not 100% sure but I guess we can try it

File: build.gradle.kts (hunk 1 above)
Comment (tso) [reply build.gradle.kts:178]: Was breaking stuff with it include + wisp doesn't have it

**Pull request merged**